### ISSUE TITLE
Criteria for absence/presence table

### ISSUE THREAD

**Issue by RaquelVaya** [Nesvilab/FragPipe-Analyst-doc#53]
Hi

Where can I find the inclusion/exclusion criteria for proteins that lead to the absence/presence table?

Thank you

Raquel

**Comment by hsiaoyi0504** [comment]
@RaquelIQS Can you elaborate a bit what's your question? It's based on the filters you defined on the left hand side menu.

**Comment by RaquelVaya** [comment]
Yes! Of course. In the Absence/Presence tab, there is a downloadable table called 'Filtered Table.' However, in the 'Filtered Options,' I only see an option to apply a filter for proteins with more than two peptides. Is this the only filter used as a criterion for excluding proteins, or are there additional filters? Are spectral counts, coverage, or protein probability considered as additional criteria for filtering proteins, or is the 'more than two peptides' filter the only one applied? Thank you very much for the attention!

Raquel

**Comment by hsiaoyi0504** [comment]
@RaquelIQS Sorry for late reply. Currently, that's the only criteria, but we could add more if you have some suggestion.

**Comment by RaquelVaya** [comment]
Hi @hsiaoyi0504,

Sorry for the delay in responding, and thank you for your reply! It would be great to consider adding Protein Probability and Top Peptide Probability as additional criteria, for example. While it’s not usually necessary, when dealing with a large number of peptides—such as in protein corona studies or cellular samples—there is a possibility of detecting proteins from other sources, and the combined peptide count alone might not be enough. Adding these filters could help improve the reliability of the results.

thanks!

Raquel
